components/bl-autocomplete-component/src/text-field/input.js

```diff
@@ -0,0 +1,55 @@
+import { forwardRef } from 'react';
+
+export const Input = forwardRef((props, ref) => {
+  const {
+    inputValue,
+    disabled,
+    autocompleteValue,
+    isOptionsOpen,
+    onChange,
+    setInputValue,
+    setIsOptionsOpen,
+  } = props;
+
+  let value;
+
+  if (!autocompleteValue) {
+    value = inputValue;
+  } else if (!inputValue && autocompleteValue) {
+    value = autocompleteValue?.label;
+  } else {
+    value = inputValue;
+  }
+
+  const handleChange = ({ target: { value } }) => {
+    setInputValue(value);
+    setIsOptionsOpen(true);
+    
+    if (onChange) {
+      onChange({ inputValue: value });
+    }
+  };
+
+  const handleClick = () => {
+    if (!disabled) {
+      setIsOptionsOpen(!isOptionsOpen);
+    }
+  };
+
+  return (
+    <div
+      onClick={ handleClick }
+      className="input__container">
+      <input
+        ref={ ref }
+        type="text"
+        id="autocomplete"
```

Comment:
Do we use it? What if we have more than one autocomplete on page?
There should be generated id for each autocomplete.